modules/post/windows/gather/hashdump.rb

```diff
@@ -145,15 +145,16 @@ def capture_hboot_key(bootkey)
         hash.update(vf[0x70, 16] + @sam_qwerty + bootkey + @sam_numeric)
 
         rc4 = OpenSSL::Cipher.new("rc4")
+        rc4.decrypt
         rc4.key = hash.digest
         hbootkey  = rc4.update(vf[0x80, 32])
         hbootkey << rc4.final
         hbootkey
```

Comment:
This works fine in Ruby 2.7 and 3.0, just updating for consistency

<details>

<summary>Code snippet</summary>

```ruby
require 'openssl'

value_data = "B" * 2000

hash = Digest::MD5.new
hash.update("hello world")

rc4 = OpenSSL::Cipher.new('rc4')
rc4.decrypt
rc4.key = hash.digest
hboot_key = rc4.update(value_data[0x80, 32])
hboot_key << rc4.final
hboot_key
```

</details>